pkg/resources/cluster_resources.go

```diff
@@ -222,4 +222,5 @@ func (c *ClusterResources) DeleteResources(resources map[string]*ResourceTracker
 
 		time.Sleep(10 * time.Second)
 	}
+
```

Comment:
nit: can we remove this change? Asking cause it keeps the PR cleaner (less files changed, etc) and easier to review.